pbs_light.go

```diff
@@ -44,16 +44,18 @@ type AccountMetrics struct {
 	RequestMeter      metrics.Meter
 	BidsReceivedMeter metrics.Meter
 	PriceHistogram    metrics.Histogram
+	AdapterMetrics    map[string]*AdapterMetrics
```

Comment:
eh... you're right. Yeah, I was going to suggest changing the type here... but on a second look, it's tied to the `exchanges` variable, which already exists and uses `string` too.

Definitely not worth going down that rabbit hole.